src/Model/Profile.php

```diff
@@ -42,15 +42,15 @@ public function getDetails() : PersonDetails
     }
 
     /**
-     * @return Sequence|Place[]
+     * @return Sequence|AccessControl[]|Place
```

Comment:
Need to lose `Place` here (unfortunately, oh for generics...)

Reply:
<img src="https://javabeanz.files.wordpress.com/2009/06/300pxjava_logo-svg.png" width=50 />
Was that a standard notation? As in, this could mean `a Sequence of Access Control object each containing a Place`

Reply:
`|` is `or`. The real return value is `Sequence`; `AccessControl[]` technically means an array of `AccessControl` (which _isn't_ correct, but is there to help IDEs); `Place` isn't returned here (and there's no way of saying that the value of `AccessControl` is a `Place`).